systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -157,6 +161,77 @@ void testKafkaConnectWithFileSinkPlugin() throws Exception {
         KafkaTopicUtils.waitForKafkaTopicDeletion(CONNECT_TOPIC_NAME);
     }
 
+    @Test
+    @Tag(NODEPORT_SUPPORTED)
+    void testKafkaConnectWithPlainAndScramShaAuthentication() throws InterruptedException, ExecutionException, TimeoutException {
+        // Use a Kafka with plain listener disabled
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3)
+                .editSpec()
+                    .editKafka()
+                        .withNewListeners()
+                            .withNewPlain()
+                                .withNewKafkaListenerAuthenticationScramSha512Auth()
+                                .endKafkaListenerAuthenticationScramSha512Auth()
+                            .endPlain()
+                            .withNewKafkaListenerExternalNodePort()
+                                .withNewKafkaListenerAuthenticationScramSha512Auth()
+                                .endKafkaListenerAuthenticationScramSha512Auth()
```

Comment:
why configuring the external node port? I don't see client or kafka connect are going to use it. Maybe it's related to the stupid question above :-)

Reply:
If you take a look at  `line 223` you will see the invocation of the client's method 

```
kafkaClient.sendAndRecvMessagesScramSha(USER_NAME, NAMESPACE, CLUSTER_NAME, CONNECT_TOPIC_NAME, 2);
```
which means, that Kafka Connect communicates with internal listener of Kafka and this client communicates with the external listener.

I hope, this makes any sense  :100: .

Reply:
Sorry but how it's clear that client is using external listener from that call? There is a username, a namespace, a cluster name and a topic name :-/

Reply:
I have got an idea but if you took a closer look on from where the client is you will find out that he is inside `externalClients package`. Anyway i can rename his for instance `externalBasicKafkaClient`. What do you think? @ppatierno

Reply:
Yeah, I think that any solution which makes clear from where the client is connecting works fine for me.

Reply:
Done.